package.json

```diff
@@ -11,14 +11,16 @@
   "license": "MIT",
   "dependencies": {
     "mustache": "^2.3.0",
-    "ncp": "^2.0.0",
```

Comment:
This file does not contain parsable JSON. Expecting ',' delimiter.

*Origin: JSONFormatBear, Section: `json`.*